src/Autosuggest.js

```diff
@@ -1020,24 +1021,47 @@ export default class Autosuggest extends React.Component {
     const noneSelected = !this.props.multiple || !this.state.selectedItems.length
     // set autoComplete off to avoid a redundant browser drop-down menu,
     // but allow it to be overridden by extra props for auto-fill purposes
-    return <input
-      autoComplete="off"
-      {...extraProps}
-      className={classNames(this.props.className,
-        { 'form-control': !this.props.multiple })}
-      ref="input"
-      key="input"
-      id={controlId}
-      disabled={this.props.disabled}
-      required={this.props.required && noneSelected}
-      placeholder={noneSelected ? this.props.placeholder : undefined}
-      type={this.props.type}
-      value={this.state.inputValue}
-      onChange={this._handleInputChange}
-      onKeyDown={this._handleKeyDown}
-      onKeyPress={this._handleKeyPress}
-      onFocus={this._handleInputFocus}
-      onBlur={this._handleInputBlur} />
+    if(this.props.type === 'textarea') {
+      return <textarea
```

Comment:
Instead of duplicating the element creation, please use a [dynamic tag](https://stackoverflow.com/questions/33471880/dynamic-tag-name-in-jsx-and-react) instead.

Reply:
Didn't see the email notification on this.  How's that change?